packages/react-progress-bar/lib/react-progress-bar.js

```diff
@@ -31,12 +31,12 @@ export const ProgressBar = ({
 	);
 
 	const loaderText = hasLabel && (
-		<span className="sui-progress-text">{'' === value ? 0 : value}%</span>
+		<span className="sui-progress-text">{isNaN(value) ? 0 : value}%</span>
```

Comment:
@creador-dev, I recommend you here to create a function that first checks if the element is `undefined`; if not, move to the following statement but return zero value if it is not defined. Then, check if the element is empty; if not, proceed to the next checkpoint, but if not, return zero value. Finally, check if the element is a number; if it is, then return the value, but if not, return zero.